### PR TITLE
feat: Add splash screen 

### LINKED ISSUE #750: Add splash screen
**Describe the feature you'd like**
Show splash screen until the fragment is ready.

**Would you like to work on the issue?**

- [x] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by liveHarshit**
Fixes #750 
Show splash screen until fragment is ready.

Screenshots for the change:
<p align="center">
 
</p>

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Not looking good. Secondly, the logo should be Open Event and not FOSSASIA

**Comment by kshitijsuri90** [comment]
Cream/ White color background with open event logo and name centered will look good.

File: app/src/main/java/org/fossasia/openevent/general/SplashActivity.kt [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import android.os.Bundle
+
+class SplashActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        startActivity(Intent(this, MainActivity::class.java))
+        finish()
+    }
+}
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/SplashActivity.kt]: No new Activity should be created

Comment (iamareebjamal) [reply app/src/main/java/org/fossasia/openevent/general/SplashActivity.kt]: https://plus.google.com/+AndroidDevelopers/posts/Z1Wwainpjhd

**Comment by liveHarshit** [comment]
@nikit19 but most of the app show splash until fragment or activity is ready. Event in Eventbrite also.

File: app/src/main/java/org/fossasia/openevent/general/SplashActivity.kt (hunk 1 above)
Comment (liveHarshit) [reply app/src/main/java/org/fossasia/openevent/general/SplashActivity.kt]: Thank you. I will follow the documentation.

**Comment by liveHarshit** [comment]
Updated. 28Px text image is used. Theme is similar to Eventbrite.

**Comment by iamareebjamal** [comment]
Logo is not correct

**Comment by liveHarshit** [comment]
> Logo is not correct

Mean text style? or logo should be there.

**Comment by nikit19** [comment]
Use the official eventyay logo

**Comment by liveHarshit** [comment]
> Use the official eventyay logo

Where can I find it in vector or svg format?

**Comment by liveHarshit** [comment]
Not available in fossasia-artwork repo.

**Comment by nikit19** [comment]
you can use png too. Isn't this png already there in the app. The one which is used in the login fragment

**Comment by iamareebjamal** [comment]
Logo should be there

**Comment by liveHarshit** [comment]
> Logo should be there

I can't find the logo for eventyay. Please help

**Comment by iamareebjamal** [comment]
Check the issue where we discussed the app icon

**Comment by liveHarshit** [comment]
Updated

**Comment by iamareebjamal** [comment]
The background should be light purple like the circle of the icon

**Comment by iamareebjamal** [comment]
And the icon is wrong

**Comment by iamareebjamal** [comment]
It's still quite dark, did you take the lower color of the circle

**Comment by liveHarshit** [comment]
> It's still quite dark, did you take the lower color of the circle

No, I take the upper half. Let me update

**Comment by liveHarshit** [comment]
For lower half. Should I update it?
<p align="center">
 
</p>

**Pull request merged**